mod-circulation/src/main/resources/domain/mod-circulation/features/loans.feature

```diff
@@ -83,3 +83,72 @@ Feature: Loans tests
     Then status 200
     And match response.itemStatusPriorToCheckIn == 'Checked out'
     And match response.itemId == itemId
+
+  Scenario: Get Loans collection
+    * def extInstanceTypeId = call uuid1
+    * def extInstitutionId = call uuid1
+    * def extCampusId = call uuid1
+    * def extLibraryId = call uuid1
+    * def extItemId1 = call uuid1
+    * def extItemId2 = call uuid1
+    * def limit = 5
```

Comment:
why do we need it ?

Reply:
extItemId1 and extItemId2 are needed to differentiate our items, since I am checking out 2 different items for one user so that I can see more items on the response, if it is superfluous action, I can just do with one item, then these definitions can also be removed